_posts/2017-11-27-visiting-clapham-south-deep-level-shelter.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Visiting Clapham South: Deep Level Shelter"
+published: 2017-11-27 21-35-27 +00:00
+tags: photography hidden-london london-transport-museum
+---
+
+A couple of weeks ago, I joined a Hidden London tour down into the tunnels
+under Clapham South tube station. They have a fascinating and much longer than
+you'd ever expect history.
```

Comment:
Feels like history is in the wrong place here, "They have a fascinating and much longer history than you'd ever expect." maybe?